### ISSUE TITLE
[Setting] SonarCloud Bug

### ISSUE THREAD

**Issue by dreamsh19** [swsnu/swpp2019-team6#101]
현재 소나클라우드 설정이 뭔가 정상적이지 않아 backend 체크를 안하고 있다던가, fail이 떴다가 다시 pass로 바뀌는 등의 이슈가 있었는데요

Travis CI의 로그를 보니 프론트엔드와 백엔드 모두 체크를하긴 하는데 소나클라우드에서 둘 중에 하나만 결과로 보여지는 것 같습니다. 현재 소나클라우드에 들어가보면 프론트엔드 소스코드만 취급하고  있네요. 

관련해서 좀 찾아보았더니 저희가 지금 backend와 frontend 소나클라우드가 2개의 머신에서 따로 돌아가고 있어서 둘 중 먼저 종료된 결과를 나중에 종료된 결과가 덮어쓰는 것 같습니다.

그래서 두 개의 머신에서 나온 결과를 합쳐서 가져오는 방법을 찾아보았는데 마땅히 좋은 방법을 찾지 못했습니다. 현재 소나클라우드 설정이 잘 되어 있는 다른 팀 레포를 확인해보니 모두 하나의 머신에서 백엔드 및 프론트엔드 소나클라우드 체크를 모두 진행하도록 세팅이 되어 있더라구요. 참고해서 저희 팀 세팅도 수정하겠습니다.

**Comment by RXWE** [comment]
![image](https://user-images.githubusercontent.com/37146511/70710905-93442900-1d23-11ea-950e-faf08c93a477.png)
그렇군요 백엔드 테스트가 먼저 돌아가고 프론트엔드 테스트가 나중에 돌아가서 프론트 결과만 보이는 거였네요... 아마 소나클라우드 설정 파일을 root에 하나만 두어서 거기에 프론트엔드 테스트랑 백엔드 테스트가 같이 설정되면 해결되지 않을까 싶어요. 수정해주셔서 감사합니다!:smile:

**Comment by deploy-soon** [comment]
네 수정해주실 때 백엔드, 프론트엔드 경로만 유의해서 해주시면 될 것 같습니다! 백엔드는 특히 한번 더 하위폴더로 되어 있어 경로를 유의해서 적어야하더라고요
